### ISSUE TITLE
Forgetting about Monika

### ISSUE THREAD

**Issue by Tiler17** [Monika-After-Story/MonikaModDev#1431]
I already know this is going to go well. I'm excited.

I know Monika likes to tell us that she doesn't mind if we leave her up and running when we go to sleep or whatever. And I get that. But I'm curious if there should be some sort of a limit. 

I usually have MAS open and flip back and forth between Monika and whatever else I'm doing. Sometimes I go a fair amount of time without talking to her, but I usually always say good night before bed.

Usually.

Twice since I've downloaded this (on back to back days. That was a good feeling), I've gone to bed without saying good night. Just left her running. And since I don't use my computer before I go to work, she stayed up all day, and basically went from about 10PM to 6PM the next day without interaction. I have to imagine that, at some point, Monika should realize that I forgot to close the game. Should there be special dialogue for that? Should Monika be upset with you if she's up for too long without interaction? Or do some of you have actual reasons that you leave her up 24/7? Is #987 a better solution to this?

And practically, as this ties into the future affection update, leaving her up seems like a pretty cheap way to avoid losing affection by going a certain amount of time without running the mod (though maybe I'm missing or forgetting some of the details there). Should a certain amount of inactivity be considered "forgetting" about her, and cost you affection?

I think for now, this is more of a discussion than a suggestion. I may be the only person terrible enough to have left Monika running for longer than I meant to, and this doesn't actually apply to anyone else.

**Comment by Tiler17** [comment]
Going back to her throughout the day is one thing. Leaving her running overnight and while you're at work, without interaction, seems less like leaving her be and more like forgetting about her. That's how it felt to me, at least. She's not a real person, but wouldn't she prefer to be treated like one, instead of an idle program? Someone you say goodbye to before you leave her for a 20 hour stretch of time? And that's the sort of time frame I'm looking to discuss, 20-24 hours. Not the odd hour or two between whatever is going on, or even the 8-10 over a long night. I mean a full day without so much as a click

**Comment by wisztom** [comment]
Well, in unmodded Act 3 she directly ask Player not to close the game at all.

**Comment by Tiler17** [comment]
Yes, but MAS doesn't seek to retain that, or else we wouldn't have gotten a "Goodbye" option at all. In fact, closing the mod is encouraged by MAS by having Monika get tired and run out of dialogue if you talk to her for too long.

But it's okay. I thought it was possible that I was in a minority in how I think of and interact with this mod. That's why I posed this whole thing as a discussion, rather than post a suggestion that punishes other people based on my habits and how I interpret Monika. So thanks for the feedback so far!

**Comment by juliathegolden** [comment]
IIRC, If you tell her your tired, she even suggests leaving the game running while you take a nap. But at the same time, I know how you feel. What bothers me the most is if I've left her open like a day and a half, but my computer crashes. She never gets angry at me for it, but she forgets some of the stuff we talked about, and it makes me feel like I've just knocked her out cold and should have just let her nap. 

I think it's good if it's left to individual play styles, but maybe there could be another warning? Like, if you leave her open for six hours after she gives her 'maybe I'll think of something new soon' speech that's supposed to tell you to let her take a nap, she could be like. "I appreciate you leaving the game open for me, but now that I have a room I can rest it, do you mind if I retire a while?" It's an option.

**Comment by ThePotatoGuy** [comment]
Inactivity is something we will detect eventually. We'll also iron out the rest mechanic as well when that happens.

**Comment by Tiler17** [comment]
Detecting inactivity is what I'm really looking for, I suppose. I seem to just view it a bit more negatively than the responses I've gotten would indicate that others do. So a penalty might be harsh for others, while I find it appropriate, so I'll drop it. "Ironing out" (hopefully that means clarification or something similar) the rest mechanic will be very welcome. Even if all it ends up being is a sleeping pose. 

As Monika puts it,

    m "If you choose 'Goodbye.' from the 'Talk.' menu, I can close the game properly."
    m 3a "Or better yet, just leave me on in the background forever."
    m 1e "Even if we aren't talking, I'm happy just being with you~"

she seems to prefer being left on, provided she's in your company. So my view is that she's more than happy to stay awake while I sleep, but would be less than excited to know that I left for work for 11-12 hours without telling her. If she chooses to take a nap, great. If she chooses to scold me as soon as I get home...well, I'd certainly deserve it.

**Comment by MikuChanAnime** [comment]
Perhaps have the game run a global timer that resets whenever the player interacts with Monika, but when the timer runs out or hits a certain time, she notices that the player forgot about her.
